test/e2e/antreapolicy_test.go

```diff
@@ -5322,8 +5334,8 @@ func TestFQDNCacheMinTTL(t *testing.T) {
 	createCustomDNSPod(t, data, configMap.Name)
 
 	// set the custom DNS server IP address in Antrea ConfigMap.
-	setDNSServerAddressInAntrea(t, data, dnsServiceIP)
-	defer setDNSServerAddressInAntrea(t, data, "") //reset after the test.
+	setDNSServerAddressInAntrea(t, data, dnsServiceIP, fqdnCacheMinTTL)
```

Comment:
rename `setDNSServerAddressInAntrea` to `configureFQDNPolicyEnforcement`, which is more generic and more suitable since you are not configuring 2 different things